src/Components/Admins/Form/index.jsx

```diff
@@ -45,97 +120,100 @@ const Form = (props) => {
 
   const onSubmit = (e) => {
     e.preventDefault();
-    if (props.isEditing) {
-      props.editAdmin(formData, props.selectedAdmin._id);
+    if (id) {
+      editAdmin(formData, selectedAdmin._id);
     } else {
-      props.addAdmin(formData);
+      addAdmin(formData);
     }
-    props.setShowForm(false);
   };
 
-  const switchButtonText = props.isEditing ? 'Update' : 'Add';
+  const switchButtonText = id ? 'Update' : 'Add';
 
   return (
-    <form className={styles.myForm} onSubmit={onSubmit}>
-      <div className={styles.divContainer}>
-        <div className={styles.inputDiv}>
-          <label>First Name</label>
-          <input
-            className={styles.input}
-            type="text"
-            name="firstName"
-            value={formData.firstName}
-            onChange={onChange}
-          />
-        </div>
-        <div className={styles.inputDiv}>
-          <label>Last Name</label>
-          <input
-            className={styles.input}
-            type="text"
-            name="lastName"
-            value={formData.lastName}
-            onChange={onChange}
-          />
-        </div>
-        <div className={styles.inputDiv}>
-          <label>DNI</label>
-          <input
-            className={styles.input}
-            type="text"
-            name="dni"
-            value={formData.dni}
-            onChange={onChange}
-          />
-        </div>
-        <div className={styles.inputDiv}>
-          <label>Phone</label>
-          <input
-            className={styles.input}
-            type="text"
-            name="phone"
-            value={formData.phone}
-            onChange={onChange}
-          />
-        </div>
-        <div className={styles.inputDiv}>
-          <label>Email</label>
-          <input
-            className={styles.input}
-            type="text"
-            name="email"
-            value={formData.email}
-            onChange={onChange}
-          />
-        </div>
-        <div className={styles.inputDiv}>
-          <label>City</label>
-          <input
-            className={styles.input}
-            type="text"
-            name="city"
-            value={formData.city}
-            onChange={onChange}
-          />
-        </div>
-        <div className={styles.inputDiv}>
-          <label>Password</label>
-          <input
-            className={styles.input}
-            type="text"
-            name="password"
-            value={formData.password}
-            onChange={onChange}
-          />
+    <>
+      <form className={styles.myForm} onSubmit={onSubmit}>
+        <div className={styles.divContainer}>
+          <div className={styles.inputDiv}>
+            <label>First Name</label>
+            <Input name="firstName" type="text" value={formData.firstName} onChange={onChange} />
+          </div>
```

Comment:
You can use the prop `textLabel` on the Input component to add a label. Input component also has a div container for both label and input so you may remove all the divs with class inputDiv. It is not necessary but it will shorten the code and make it more readable. The same goes for all the others inputs.